src/common/containers/repository-setup.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component, PropTypes } from 'react';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+import { createWebhook } from '../actions/webhook';
+import { Message } from '../components/forms';
+import Spinner from '../components/spinner';
+
+import styles from './container.css';
+
+class RepositorySetup extends Component {
```

Comment:
Indeed, that was pretty much my reasoning.  The setup page will shortly also start triggering initial builds.